components/variations/BUILD.gn

```diff
@@ -130,56 +139,56 @@ static_library("test_support") {
   ]
 }
 
-source_set("unit_tests") {
-  testonly = true
-  sources = [
-    "active_field_trials_unittest.cc",
-    "caching_permuted_entropy_provider_unittest.cc",
-    "child_process_field_trial_syncer_unittest.cc",
-    "entropy_provider_unittest.cc",
-    "experiment_labels_unittest.cc",
-    "hashing_unittest.cc",
-    "net/variations_command_line_unittest.cc",
-    "net/variations_http_headers_unittest.cc",
-    "study_filtering_unittest.cc",
-    "synthetic_trial_registry_unittest.cc",
-    "variations_associated_data_unittest.cc",
-    "variations_crash_keys_unittest.cc",
-    "variations_http_header_provider_unittest.cc",
-    "variations_id_collection_unittest.cc",
-    "variations_request_scheduler_unittest.cc",
-    "variations_seed_processor_unittest.cc",
-    "variations_seed_simulator_unittest.cc",
-    "variations_seed_store_unittest.cc",
-  ]
+if (!use_cobalt_customizations) {
+  source_set("unit_tests") {
+    testonly = true
+    sources = [
+      "active_field_trials_unittest.cc",
+      "caching_permuted_entropy_provider_unittest.cc",
+      "child_process_field_trial_syncer_unittest.cc",
+      "entropy_provider_unittest.cc",
+      "experiment_labels_unittest.cc",
+      "hashing_unittest.cc",
+      "net/variations_command_line_unittest.cc",
+      "net/variations_http_headers_unittest.cc",
+      "study_filtering_unittest.cc",
```

Comment:
`study_filtering.cc` above is compiled in, we should also enable the unit tests - same for other modules above. Let's add a follow-up for enabling relevant tests.

Reply:
Done. Added TODO